packages/graphql/src/resolvers/IdentityEvents.js

```diff
@@ -79,9 +80,15 @@ export function identity({ id, ipfsHash }) {
     if (!contracts.identityEvents.options.address || !id) {
       return null
     }
+    let accounts = id
     if (!ipfsHash) {
+      const owner = await proxyOwner(id)
+      if (owner) {
+        accounts = [id, owner]
+      }
+
       const events = await contracts.identityEvents.eventCache.getEvents({
-        account: id
+        account: accounts
```

Comment:
I haven't paid close attention to the EventCache interface. I assume it expects addresses checksummed and both id and owner are indeed checksummed ?

Reply:
Yep they are

Reply:
sweet